### PR TITLE
dockerfiles for new april release

### PR DESCRIPTION
**Pull request by ironstas1k**

### CODE REVIEW

File: 11.0.7-11.39.15/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:bionic
+
```
Comment (jorsol) [inline 11.0.7-11.39.15/Dockerfile:2]: UTF-8 by default is not longer activated in this image

File: 13.0.3-13.31.11/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:bionic
+
```
Comment (jorsol) [inline 13.0.3-13.31.11/Dockerfile:2]: UTF-8 by default is not longer activated in this image

File: 14.0.1-14.28.21/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:bionic
+
```
Comment (jorsol) [inline 14.0.1-14.28.21/Dockerfile:2]: UTF-8 by default is not longer activated in this image

File: 7u262-7.38.0.11/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:bionic
+
```
Comment (jorsol) [inline 7u262-7.38.0.11/Dockerfile:2]: UTF-8 by default is not longer activated in this image

File: 8u252-8.46.0.19/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:bionic
+
```
Comment (jorsol) [inline 8u252-8.46.0.19/Dockerfile:2]: UTF-8 by default is not longer activated in this image

File: debian/8u252-8.46.0.19/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,13 @@
+FROM debian:buster
+
```
Comment (jorsol) [inline debian/8u252-8.46.0.19/Dockerfile:2]: UTF-8 by default is not longer activated in this image

File: debian/7u262-7.38.0.11/Dockerfile [hunk 7]
```diff
@@ -0,0 +1,13 @@
+FROM debian:buster
+
```
Comment (jorsol) [inline debian/7u262-7.38.0.11/Dockerfile:2]: UTF-8 by default is not longer activated in this image

File: debian/14.0.1-14.28.21/Dockerfile [hunk 8]
```diff
@@ -0,0 +1,13 @@
+FROM debian:buster
+
```
Comment (jorsol) [inline debian/14.0.1-14.28.21/Dockerfile:2]: UTF-8 by default is not longer activated in this image

File: debian/13.0.3-13.31.11/Dockerfile [hunk 9]
```diff
@@ -0,0 +1,13 @@
+FROM debian:buster
+
```
Comment (jorsol) [inline debian/13.0.3-13.31.11/Dockerfile:2]: UTF-8 by default is not longer activated in this image

File: debian/11.0.7-11.39.15/Dockerfile [hunk 10]
```diff
@@ -0,0 +1,13 @@
+FROM debian:buster
+
```
Comment (jorsol) [inline debian/11.0.7-11.39.15/Dockerfile:2]: UTF-8 by default is not longer activated in this image

File: alpine/8u252-8.46.0.19-jre-headless/Dockerfile [hunk 11]
```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.10
+
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+RUN wget --quiet https://cdn.azul.com/public_keys/alpine-signing@azul.com-5d5dc44c.rsa.pub -P /etc/apk/keys/ && \
+    echo "https://repos.azul.com/zulu/alpine" >> /etc/apk/repositories && \
+    apk --no-cache add zulu8-jre-headless
```
Comment (sjaakvandenberg-cleverbase) [inline alpine/8u252-8.46.0.19-jre-headless/Dockerfile:9]: @ironstas1k @sgrinev In previous versions ([8u232-8.42.0.23](https://github.com/zulu-openjdk/zulu-openjdk/blob/master/alpine/8u232-8.42.0.23-jre-headless/Dockerfile), [8u242-8.44.0.11](https://github.com/zulu-openjdk/zulu-openjdk/blob/master/alpine/8u242-8.44.0.11-jre-headless/Dockerfile), package versions were pinned. Was this change intentional?
